### DISCUSSION TITLE
regarding settings in `HOMEBREW_CASK_OPTS`, single use CLI commands for specific Casks, `brew install -help`

### DISCUSSION THREAD

**Discussion by 101Dude** [Homebrew/discussions#5102] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.6-64-g3bf9e72
ORIGIN: https://github.com/Homebrew/brew
HEAD: 3bf9e72c288e2348365c88e95afd626a5adefea3
Last commit: 2 hours ago
Core tap HEAD: 4b876ab401eb0c26d628091c06bd7579ecdb0d24
Core tap last commit: 11 days ago
Core tap JSON: 01 Feb 19:53 UTC
Core cask tap HEAD: a33f19d841713580333571b335c702ef0a2c9332
Core cask tap last commit: 11 days ago
Core cask tap JSON: 01 Feb 19:53 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: ["--greedy", "--require-sha", "--no-quarantine", "--appdir=/Applications", "--fontdir=/Library/Fonts", "--qlplugindir=/Library/QuickLook", "--internet_plugindir=/Library/Internet Plug-Ins", "--prefpanedir=/Library/PreferencePanes/", "--servicedir=/Library/Services"]
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.r6CeSDXxmp/org.xquartz:0
HOMEBREW_EDITOR: nano
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 16
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.1.4 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: 16-core 64-bit kabylake
Clang: 15.0.0 build 1500
Git: 2.43.0 => /usr/local/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 14.3-x86_64
CLT: 15.1.0.0.1.1700200546
Xcode: 15.2
```

### Output of `brew doctor`

```shell
14:07:19.592 at iMac191 ~ brew doctor
Your system is ready to brew.
```

### Description of issue

Regarding settings in `HOMEBREW_CASK_OPTS` and single use CLI commands for specific Casks.

I am aware of efforts to create an Anti `--require-sha` option like `--quarantine` appears to have with `--no-quarantine` _(which I would applaud and vote for_ 👍🏻)

However, since that has been voted down ([issue was closed](https://github.com/Homebrew/homebrew-cask/issues/147305)) can we at least have better documentation in `brew install -help` for `--force`?

Maybe explain that it will ignore `--require-sha`. That way uses don't have to hop onto GitHub to figure it out via various closed issues or fiddle with `HOMEBREW_CASK_OPTS` for one download.

No sarcasm intended.

Thank you.

### Relevant casks

chromium

**Comment by SMillerDev** [comment]
> can we at least have better documentation in brew install -help for --force?

We certainly can, I'm looking forward to your pull request.

**Reply by 101Dude** [reply]
Believe me you don't want me attempting to do a "pull request", lol, you'd have to come up with far more obnoxious comments and it would take up far more of your time.

I did take the time to create a discussion of an oversight in the hopes it would benefit other users. That's what I can do right now.
